File: Nx3CS/bin/test/src/nx3/qs/ParserEvents.cs

#pragma warning disable 109, 114, 219, 429, 168, 162
namespace nx3.qs{
	public  class ParserEvents : global::haxe.lang.Enum {
		static ParserEvents() {
			#line 9 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
			global::nx3.qs.ParserEvents.constructs = new global::Array<object>(new object[]{"SetGuessOctave", "SetOctave", "SetNoteVal", "SetMode", "SetBarClef"});
		}
		public    ParserEvents(global::haxe.lang.EmptyObject empty) : base(global::haxe.lang.EmptyObject.EMPTY){
			unchecked {
			}
			#line default
		}
		
		
		public    ParserEvents(int index, global::Array<object> @params) : base(index, @params){
			unchecked {
			}
			#line default
		}
		
		
		public static  global::Array<object> constructs;
		
		public static   global::nx3.qs.ParserEvents SetGuessOctave(bool mode){
			unchecked {
				#line 11 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((int) (0) ), ((global::Array<object>) (new global::Array<object>(new object[]{mode})) ));
			}
			#line default
		}
		
		
		public static   global::nx3.qs.ParserEvents SetOctave(int octave){
			unchecked {
				#line 12 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((int) (1) ), ((global::Array<object>) (new global::Array<object>(new object[]{octave})) ));
			}
			#line default
		}
		
		
		public static   global::nx3.qs.ParserEvents SetNoteVal(global::nx3.ENoteVal @value){
			unchecked {
				#line 13 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((int) (2) ), ((global::Array<object>) (new global::Array<object>(new object[]{@value})) ));
			}
			#line default
		}
		
		
		public static   global::nx3.qs.ParserEvents SetMode(global::nx3.qs.ContentMode mode){
			unchecked {
				#line 14 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((int) (3) ), ((global::Array<object>) (new global::Array<object>(new object[]{mode})) ));
			}
			#line default
		}
		
		
		public static   global::nx3.qs.ParserEvents SetBarClef(global::nx3.EClef clef){
			unchecked {
				#line 15 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((int) (4) ), ((global::Array<object>) (new global::Array<object>(new object[]{clef})) ));
			}
			#line default
		}
		
		
		public static  new object __hx_createEmpty(){
			unchecked {
				#line 9 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((global::haxe.lang.EmptyObject) (global::haxe.lang.EmptyObject.EMPTY) ));
			}
			#line default
		}
		
		
		public static  new object __hx_create(global::Array arr){
			unchecked {
				#line 9 "F:\\nx3\\src\\nx3\\qs\\ParserEvents.hx"
				return new global::nx3.qs.ParserEvents(((int) (global::haxe.lang.Runtime.toInt(arr[0])) ), ((global::Array<object>) (global::Array<object>.__hx_cast<object>(((global::Array) (arr[1]) ))) ));
			}
			#line default
		}
		
		
	}
}

File: Playground/TestDiff/java/src/nx3/qs/ParserEvents.java
package nx3.qs;
import haxe.root.*;

@SuppressWarnings(value={"rawtypes", "unchecked"})
public  class ParserEvents extends haxe.lang.Enum
{
	static 
	{
		nx3.qs.ParserEvents.constructs = new haxe.root.Array<java.lang.String>(new java.lang.String[]{"SetGuessOctave", "SetOctave", "SetNoteVal", "SetMode", "SetBarClef"});
	}
	public    ParserEvents(int index, haxe.root.Array<java.lang.Object> params)
	{
		super(index, params);
	}
	
	
	public static  haxe.root.Array<java.lang.String> constructs;
	
	public static   nx3.qs.ParserEvents SetGuessOctave(boolean mode)
	{
		return new nx3.qs.ParserEvents(((int) (0) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{mode})) ));
	}
	
	
	public static   nx3.qs.ParserEvents SetOctave(int octave)
	{
		return new nx3.qs.ParserEvents(((int) (1) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{octave})) ));
	}
	
	
	public static   nx3.qs.ParserEvents SetNoteVal(nx3.ENoteVal value)
	{
		return new nx3.qs.ParserEvents(((int) (2) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{value})) ));
	}
	
	
	public static   nx3.qs.ParserEvents SetMode(nx3.qs.ContentMode mode)
	{
		return new nx3.qs.ParserEvents(((int) (3) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{mode})) ));
	}
	
	
	public static   nx3.qs.ParserEvents SetBarClef(nx3.EClef clef)
	{
		return new nx3.qs.ParserEvents(((int) (4) ), ((haxe.root.Array<java.lang.Object>) (new haxe.root.Array<java.lang.Object>(new java.lang.Object[]{clef})) ));
	}
	
	
}


